src/components/Button.tsx

```diff
@@ -0,0 +1,34 @@
+import { FontType } from 'styled-components'
+import { BaseButton } from './styles/Button.styled'
+
+type Props = {
+  text: string
+  hasActive?: boolean
+  link?: string | undefined
+  htmlType?: 'button' | 'submit' | 'reset' | undefined
+  fontType: FontType
+}
+
+const defaultProps = {
+  hasActive: false,
+  link: null,
```

Comment:
```suggestion
  link: undefined,
```
not critical but just pointing out haha, use `undefined` instead of `null`, cos in this context `undefined` makes nore sense as `undefined` is used for the absence of value i.e. `link` param not specifed
[can read more abt null vs undefined here](https://www.geeksforgeeks.org/undefined-vs-null-in-javascript/)

Reply:
Resolved in commit 8cb1f2e3fa2ff50e16fa4367e6681ec6e97021c9.